tests/Graph/Trees/BallTreeTest.php

```diff
@@ -83,6 +85,32 @@ public function growNeighborsRange() : void
         $this->assertCount(1, array_unique($labels));
     }
 
+    /**
+     * @test
+     */
+    public function growWithRepetitions() : void
+    {
+        $size = 25;
```

Comment:
If this variable does not change can we make it a class constant? And perhaps a little more descriptive of the data it holds such as DATASET_SIZE or NUM_SAMPLES?

Reply:
Yes, I saw you did this - looks good to me, I just didn't want to intervene too much with other tests and especially generators - I thought blob generator does not accept stdDev param equal to zero and that's why I had to generate the repetitions myself. However, your solution with using existing generator is much better, I agree.